Cargo.toml

```diff
@@ -44,3 +49,8 @@ version = "1.17.0"
 features = ["fs", "io-util", "rt", "sync", "time"]
 
 [features]
+default = ["isahc"]
```

Comment:
Features should be additive. There shouldn't be any default features.

Reply:
The lack of a default would mean the crate won't compile without an explicit feature, is that OK or did you mean to always have `isahc` support not locked behind a feature?

Reply:
It's a best practice for handling features in Cargo crates. Features should always be additive, and default features should never conflict with extra features. It should be a compiler error.

Reply:
Should be addressed with the latest commit